frontend/components/Domain/Recipe/RecipeLastMade.vue

```diff
@@ -158,23 +204,50 @@ export default defineComponent({
       }
 
       lastMadeReady.value = true;
+
+      // If lastMade exists, init the date picker to that date, else set to 'today'
+      lastMadeEdit.value = lastMade.value
+        ? new Date(lastMade.value).toISOString().substring(0, 10)
+        : new Date().toISOString().substring(0, 10);
```

Comment:
Can you just do `new Date(lastMade.value).toISOString().substring(0, 10)` ; because if it's null it'd back back to current date? (or pull the bit that is different - last edited || now) out to DRY?